pkg/gateway/vald/handler/grpc/handler.go

```diff
@@ -217,44 +209,46 @@ func (s *server) search(ctx context.Context, cfg *payload.Search_Config,
 			}
 			return res, nil
 		case dist := <-dch:
-			if len(res.GetResults()) >= num {
-				if dist.GetDistance() < math.Float32frombits(atomic.LoadUint32(&maxDist)) {
-					atomic.StoreUint32(&maxDist, math.Float32bits(dist.GetDistance()))
-				} else {
-					continue
-				}
+			rl := len(res.GetResults())
```

Comment:
please follow the coding style, https://github.com/vdaas/vald/blob/master/docs/contributing/coding-style.md#variables-and-constant
```suggestion
			rl := len(res.GetResults()) // result length
```